extensions/scarb-doc/src/docs_generation/markdown/traits.rs

```diff
@@ -325,11 +337,30 @@ fn generate_markdown_from_item_data(
         writeln!(&mut markdown, "{doc}\n")?;
     }
 
-    writeln!(
-        &mut markdown,
-        "Fully qualified path: {}\n",
-        get_linked_path(doc_item.full_path())
-    )?;
+    let doc_as_any = doc_item as &dyn Any;
+    let full_path = {
+        if doc_as_any.is::<Variant>() || doc_as_any.is::<Member>() {
```

Comment:
please share, if you can come up with more idiomatic way to distinct doc_item type here 🙏